INSTALLING.md

```diff
@@ -24,10 +24,16 @@ If you are running a distribution which does not match exactly a repository uplo
 
 If you are running LinuxMint 17.1 Rebecca, which is downstream of Ubuntu Trusty and Debian Jessie, you can run:
 
-`curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | os=debian dist=jessie sudo -E sudo bash`
+`curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | os=debian dist=jessie sudo -E bash`
 
 The `os` and `dist` variables passed-in will override what would be detected for your system and force the selection of the upstream distribution's repository.
 
+You may also be able to run the following to automatically detect the dist for Ubuntu based distributions such as Pop OS:
+```
+. /etc/lsb-release
```

Comment:
This line is going to import everything in that file into the current shell, which may not be what people want.  Could we use parentheses around these two commands to import that in a subshell?  I'd also recommend in general avoiding `sudo -E` because sometimes that functionality is disabled, so maybe we'd end up with something like this:

```
(. /etc/lsb-release &&
curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | 
sudo env os=ubuntu dist="${DISTRIB_CODENAME}" bash)
```

Would that work for you?

Reply:
Yep, that looks much better. Will update the PR